docs/docs/Components/components-data.mdx

```diff
@@ -171,6 +171,24 @@ You can toggle parameters through the <Icon name="SlidersHorizontal" aria-hidden
 | ignore_unspecified_files | Ignore Unspecified Files | Input parameter. If true, `Data` with no `file_path` property is ignored. If false (default), the component errors when a file isn't specified. |
 | concurrency_multithreading | Processing Concurrency | Input parameter. The number of files to process concurrently if multiple files are uploaded. Default is 1. Values greater than 1 enable parallel processing for 2 or more files. |
 
+### Advanced parsing
+
+As of Langflow 1.6, the File component supports advanced document parsing using the [Docling](https://docling-project.github.io/docling/) library.
+Only non-tabular documents can be parsed, and only one document at a time.
+The parser requires the Docling dependency to be installed. As of Langflow 1.6, the Docling dependency is included with Langflow.
```

Comment:
NIT: do we want to standardize on the casing of docling? (tbh i dont actually know which one is "correct")

Reply:
Yes good call out. It should be capitalized everywhere as Docling.

<img width="775" height="144" alt="image" src="https://github.com/user-attachments/assets/47e2a4e0-fa6c-4227-8551-e2706abc33bc" />

Lowercase should only be used in code where the syntax rules use lowercase, like `docling`.